code/modules/surgery/surgery_helpers.dm

```diff
@@ -11,6 +11,17 @@
 	else
 		return 0.5
 
+/proc/get_organmanip_location_isopened(mob/living/carbon/located_mob, location)
```

Comment:
this needs a better name and some dmdocing https://github.com/tgstation/tgstation/blob/master/.github/guides/AUTODOC.md

Reply:
How is this: 
/proc/isLocationOpened